src/assets/src/components/queue.tsx

```diff
@@ -139,61 +140,63 @@ const WaitingTurnAlert = (props: WaitingTurnAlertProps) => {
 
 
 interface VideoMeetingInfoProps {
-    meetingBackend: MeetingBackend;
     metadata: BluejeansMetadata | ZoomMetadata;
+    backend: MeetingBackend;
 }
 
 const VideoMeetingInfo: React.FC<VideoMeetingInfoProps> = (props) => {
-    const joinLink = props.metadata.meeting_url
-        ? (
-            <Button as='a' href={props.metadata.meeting_url} target='_blank' variant='warning' className='mr-3'>
-                Join Meeting
-            </Button>
-        )
-        : <p><strong>Please wait. The meeting is being started...</strong></p>;
-
     const docLinkTag = (
         <a
-            href={props.meetingBackend.docs_url === null ? undefined : props.meetingBackend.docs_url} 
+            href={props.backend.docs_url === null ? undefined : props.backend.docs_url}
             target='_blank'
             className='card-link'
         >
-            How to use {props.meetingBackend.friendly_name} at U-M
+            How to use {props.backend.friendly_name} at U-M
         </a>
     );
 
-    const extraDetails = props.meetingBackend.name === 'bluejeans' && (
-        <Col md={6} sm={true}>
-            <Card>
-                <Card.Body>
-                    <Card.Title className='mt-0'>Having Trouble with Video?</Card.Title>
-                    <Card.Text>
-                        <BlueJeansDialInMessage meetingNumber={props.metadata.numeric_meeting_id} />
-                        You are not a moderator, so you do not need a moderator passcode.
-                    </Card.Text>
-                </Card.Body>
-            </Card>
-        </Col>
-    );
+    let dialInMessage;
+    if (props.metadata.numeric_meeting_id) {
+        const dialInProps = {
+            phone: props.backend.telephone_num,
+            meetingNumber: props.metadata.numeric_meeting_id,
+            intlNumbersURL: props.backend.intl_telephone_url
+        } as DialInMessageProps;
+
+        dialInMessage = props.backend.name === 'zoom'
+            ? <ZoomDialInMessage {...dialInProps} />
+            : props.backend.name === 'bluejeans'
+                ? <BlueJeansDialInMessage {...dialInProps} />
+                : null;
+    }
```

Comment:
I made a decision here, but I'm happy to walk it back. The "Having Trouble with Video?" card just doesn't seem that useful until the meeting has been created (i.e. it has a meeting number). Perhaps there is alternative text we could use before the meeting is created...

Reply:
Seems to make sense to me.

Reply:
Cool, I will ask for more feedback during the QA process.